frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnFieldFooter.tsx

```diff
@@ -20,7 +20,7 @@ const MultiColumnFieldFooter: React.FC<MultiColumnFieldHeader> = ({
     icon={<PlusCircleIcon />}
     isInline
   >
-    {addLabel || 'Add'}
+    {addLabel || 'Add values'}
```

Comment:
how about `Add value` ?

Reply:
I reverted the change where you [removed this value](https://github.com/openshift/console/pull/5998/files#diff-d4751c6c009a3bb5e09991b50a9d98afL23). I didn't customize the text.